src/storage/sftp/storage.c

```diff
@@ -897,6 +1142,120 @@ storageSftpNew(
                     fingerprint, strZ(param.hostFingerprint));
             }
         }
+        else if (param.sftpStrictHostKeyChecking == SFTP_STRICT_HOSTKEY_CHECKING_NONE)
+        {
+            LOG_WARN_FMT("host key checking disabled (repo-sftp-host-key-check-type=none), connections are not secure!");
```

Comment:
Why did we add this warning? We don't want to emit warnings that the user has no ability to stop. This is not the default setting so they made a conscious choice -- not our problem.

Reply:
I removed this in fd992dd9fbb9e.